eth/api.go

```diff
@@ -346,6 +347,47 @@ func (api *PrivateDebugAPI) GetBadBlocks(ctx context.Context) ([]*BadBlockArgs,
 	return results, nil
 }
 
+type addressMap map[common.Hash]common.Address
+
+type AccountRangeResult struct {
+	AddressMap addressMap `json:"addressMap"`
+}
+
+func accountRange(st state.Trie, start *common.Hash, maxResult int) (AccountRangeResult, error) {
```

Comment:
I don't really see the value of wrapping this into a custom type versus just returning `map[common.Hash]common.Address` in the first place.

Reply:
The part not implemented here is that also supposed to return a `nextKey` field 😢